Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -2522,7 +2573,11 @@ DataProcessorSpec getAODProducerWorkflowSpec(GID::mask_t src, bool enableSV, boo
   outputs.emplace_back(OutputLabel{"O2ambiguousMFTtrack"}, "AOD", "AMBIGUOUSMFTTR", 0, Lifetime::Timeframe);
   outputs.emplace_back(OutputLabel{"O2ambiguousFwdtrack"}, "AOD", "AMBIGUOUSFWDTR", 0, Lifetime::Timeframe);
   outputs.emplace_back(OutputLabel{"O2v0_001"}, "AOD", "V0_001", 0, Lifetime::Timeframe);
+#ifdef O2_ZDC_NEWDATAMODEL
+  outputs.emplace_back(OutputLabel{"O2zdc_001"}, "AOD", "ZDC_001", 0, Lifetime::Timeframe);
```

Comment:
This is incorrect, it should be `"ZDC", 1`, the version for AOD tables is encoded in subspec.

Reply:
OK, this is wrong for all of them. See v0, cascade, collision etc. So it seems to work but is not good of course...

Reply:
Shall we change this everywhere? Or we keep doing as is? What do you think?

Reply:
Probably better to do the rest in a separate PR, for bookkeeping reasons.

Reply:
You mean the rest or all of them? What do you think? :-D

Reply:
ZDC should be done correctly from the start here, I think. The rest can be fixed elsewhere so we do not mix them with ZDC-specific change.

Reply:
Okay, the latest PR does as you suggested, @aalkin ! Can you please check if I did no do anything major wrong? (technically, though, it's in the part of the producer that is as-of-yet disabled, but I want to be sure it's ok already). If this is fine I would suggest we go ahead and merge so I can do a PR for the O2Physics part. Thanks!!!